### ISSUE TITLE
Unable to identify scroll bar "Page down" or other internal scroll bar object using UIAutomation

### ISSUE THREAD

**Issue by preetipadalia** [dotnet/wpf#3597]
* .NET Core Version: (e.g. 3.0 Preview1, or daily build number, use `dotnet --info`): 3.0.1.302
* Windows version: (`winver`): Windows 10/Windows Server 2012/ Windows Server 2016
* Does the bug reproduce also in WPF for .NET Framework 4.8?: No
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? No
* Security issues and bugs should be reported privately, learn more via our [responsible disclosure guidelines](https://github.com/dotnet/wpf/blob/master/README.md#reporting-security-issues-and-security-bugs).

 
 
 **Problem description:**
 
We are automating windows Desktop applications using UIAutomation library. We were using net472 previously and now switched to .net core 3.1.

csproj file entries:

<?xml version="1.0" encoding="utf-8"?> 
<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop"> 
  <PropertyGroup> 
    <OutputType>Library</OutputType> 
    <TargetFrameworks>netcoreapp3.1</TargetFrameworks> 
    <UseWPF>true</UseWPF> 
    <UseWindowsForms>true</UseWindowsForms>
We are using the following code to identify the objects at runtime using UIAutomation library.

using System.Windows.Automation;
PropertyCondition condition=new PropertyCondition(AutomationElement.NameProperty, “nameOfObject”)
AutomationElementCollection collection=windowElement.FindAll(TreeScope.Descendants, condition); 
Most of the objects are working but the objects inside the scroll bar are not identifiable using code.
[WindowsFormsApp1.zip](https://github.com/dotnet/wpf/files/5303322/WindowsFormsApp1.zip)
[NetCoreSample.zip](https://github.com/dotnet/wpf/files/5303324/NetCoreSample.zip)
[netframeworkCode.zip](https://github.com/dotnet/wpf/files/5303325/netframeworkCode.zip)

**Comment by ryalanms** [comment]
@preetipadalia:  Are you using WPF or only WinForms?  Thanks.

**Comment by preetipadalia** [comment]
I am using following code to retrieve information of objects : 
```
using UIAutomation library.
using System.Windows.Automation;
PropertyCondition condition=new PropertyCondition(AutomationElement.NameProperty, “nameOfObject”)
AutomationElementCollection collection=windowElement.FindAll(TreeScope.Descendants, condition);
```
I am able to get information of most of the objects except scroll bar. I am not using any wpf component.

**Comment by preetipadalia** [comment]
@ryalanms any update on this?

**Comment by anjali-wpf** [comment]
@preetipadalia Could you please share more information on how to reproduce this, a screen recording would be helpful in order to understand it better.
